CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # did:key driver ChangeLog
 
+## 0.6.1 - TBD
+
+### Changed
+- Return public/private key pair from `generate()`, available on `didDoc.keys`.
+  Example: `const didDoc = await driver.generate(); didDoc.keys // -> keys obj`.
```

Comment:
I'm -1 to putting code in the changelog like this, also you updated the readme.
```suggestion
```

Reply:
@mattcollier no prob, removing.